babel/messages/frontend.py

```diff
@@ -825,6 +882,7 @@ def _configure_command(self, cmdname, argv):
             strs = ["--%s" % name]
             if short:
                 strs.append("-%s" % short)
+            strs.extend(cmdclass.option_aliases.get(name, ()))
```

Comment:
Oop, good catch.

The alias mapping should be `'output_file': ('--output',)`...